build.sh

```diff
@@ -491,12 +492,20 @@ if (( ${NUMARGS} == 0 )) || hasArg go; then
 fi
 
 # Build the cuvs Java bindings
+RUN_JAVA_TESTS=""
+for arg in "$@"
+do
+  if [ "$arg" == "--run-java-tests" ]; then
+    RUN_JAVA_TESTS="--run-java-tests"
+  fi
+done
+
 if (( ${NUMARGS} == 0 )) || hasArg java; then
     if ! hasArg libcuvs; then
         echo "Please add 'libcuvs' to this script's arguments (ex. './build.sh libcuvs java') if libcuvs libraries are not already built"
     fi
     cd ${REPODIR}/java
-    ./build.sh
+    ./build.sh $RUN_JAVA_TESTS
```

Comment:
```suggestion
    if hasArg --run-java-tests; then
        ./build.sh --run-java-tests
    else
        ./build.sh
     fi
```

I think this is simpler than introducing a new variable (`RUN_JAVA_TESTS`) and a ` for` loop over all arguments. Would you please consider it?

Reply:
fixed